modules/Core/src/main/java/org/terasology/logic/inventory/InventoryUIClientSystem.java

```diff
@@ -24,17 +26,37 @@
 import org.terasology.entitySystem.systems.RegisterSystem;
 import org.terasology.input.ButtonState;
 import org.terasology.input.binds.inventory.InventoryButton;
+import org.terasology.logic.characters.CharacterComponent;
 import org.terasology.logic.characters.interactions.InteractionUtil;
+import org.terasology.logic.players.LocalPlayer;
 import org.terasology.network.ClientComponent;
+import org.terasology.protobuf.EntityData;
+import org.terasology.registry.CoreRegistry;
 import org.terasology.registry.In;
 import org.terasology.rendering.nui.ControlWidget;
 import org.terasology.rendering.nui.NUIManager;
+import org.terasology.rendering.nui.layers.ingame.inventory.InventoryCell;
+
+import java.util.ArrayList;
+import java.util.List;
 
 @RegisterSystem(RegisterMode.CLIENT)
 public class InventoryUIClientSystem extends BaseComponentSystem {
+    
+    EntityRef movingItemItem = EntityRef.NULL;
+    
+    int movingItemCount = 0;
```

Comment:
Here too. Access modifiers help clarity :-) You can also javadoc these fields just for a quick one-liner about what they're for (ideally using the single line `/** Some description */` format)